pages/productListingPage/productListingPage-vert.css

```diff
@@ -0,0 +1,95 @@
+@import "/styles.css";
+
+:root {}
+
+body {}
+
+.page-wrap {
+    grid-template-areas:
+        'p_nav p_nav p_nav p_nav p_nav p_nav'
+        'p_sidebar p_sidebar p_content p_content p_content p_content'
+        'p_sidebar p_sidebar p_content p_content p_content p_content';
+}
+
+/* main content */
+.main-content {
+    width: 100%;
+    height: fit-content;
+    align-items: center;
+    display: grid;
+    gap: 2rem;
+    justify-content: center;
+    grid-template-columns: repeat(auto-fit, clamp(27rem, 32%, 35rem));
+}
+
+/* footer */
+.page-footer {
+    display: none;
+}
+
+/* Overriding existing property from library */
+.dui-inp-slider {
+    width: 80%;
+}
+
+.mobl-fltr-btn-helper {
+    width: 100%;
+    height: 7rem;
+    display: none
+}
+
+.mobl-fltr-btn {
+    width: 100%;
+    height: 7rem;
+    background-color: var(--dui-light-theme-bg);
+    box-shadow: 0px 0px 0px 2px var(--dui-primary-color-p2-lgt-35pct);
+
+    position: fixed;
+    bottom: 0;
+    left: 0;
+
+    display: flex;
```

Comment:
make helper class for flex

Reply:
I have made a choice of not using class for single CSS property.